services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluTormaysService.kt

```diff
@@ -168,9 +168,9 @@ enum class TormaystarkasteluBussiRunkolinja(val runkolinja: String) {
 
     fun toLinjaautoliikenneluokittelu(): Linjaautoliikenneluokittelu =
         when (this) {
-            ON -> Linjaautoliikenneluokittelu.RUNKOLINJA
-            LAHES -> Linjaautoliikenneluokittelu.RUNKOLINJAMAINEN
-            EI -> Linjaautoliikenneluokittelu.PERUS
+            ON -> Linjaautoliikenneluokittelu.RUNKOLINJA_TAI_ENINTAAN_20_VUOROA_RUUHKATUNNISSA
+            LAHES -> Linjaautoliikenneluokittelu.ENINTAAN_10_VUOROA_RUUHKATUNNISSA
+            EI -> Linjaautoliikenneluokittelu.EI_RUUHKAAIKANA
```

Comment:
This doesn't make much sense anymore.

Reply:
How so?

Reply:
Why does being almost a trunk line lead to having 10 buses?
```kotlin
LAHES -> Linjaautoliikenneluokittelu.ENINTAAN_10_VUOROA_RUUHKATUNNISSA
```

Does this mean this is not a trunk line at rush hour?
```kotlin
EI -> Linjaautoliikenneluokittelu.EI_RUUHKAAIKANA
```

Reply:
This is mostly from old implementation but there should not be much changes to it. New specification says:
```
2 = "Linja-autojen paikallisliikennettä muulloin, kuin ruuhka-aikana"
3 = "Linja-autojen paikallisliikenteen linjojen vuoromäärä enintään 10 ruuhkatunnissa"
4 = "Linja-autojen paikallisliikenteen runkolinja tai vuoromäärä enintään 20 ruuhkatunnissa"
```
That `when` logic follows this specification in my opinion: 2 means that there is no "runkolinja", 3 means that there is "almost" some "runkolinja" and "yes" means that there is "runkolinja".

Reply:
I can't find anything about the "almost" trunk lines from the new specification. Collecting those values seems to have been removed from the material update scripts. Maybe we can remove the `LAHES` value completely?

Reply:
You are correct. I removed "almost".